test/test.js

```diff
@@ -9,7 +9,7 @@ describe("loader", function() {
 			query: "?abc.def.ghi=>1"
 		}, "").should.be.eql(HEADER +
 			"var abc = (abc || {});\n" +
-			"abc.def = {};\n" +
+			"abc.def = abc.def || {};\n" +
```

Comment:
It does seem like there should be a test that is not impacted by this change. That probably means adding a test for the case where there is no nesting, or exec the generated code and test its behavior:

```javascript
describe("when the import is nested", function(){
   it("creates the nested variable", function(){
....
   });
   describe ("when the import is a plugin to an existing library", function(){
        it ("adds the functionality to the library", function (){ 
            expect(some.nested.library.originalProperty).toBeDefined ()
...
```

That would be a larger refactor to how the tests are structured.